raiden/tests/scenarios/bf1_basic_functionality.yaml

```diff
@@ -66,129 +69,141 @@ scenario:
             - assert: {from: 4, to: 0, total_deposit: 1000, balance: 1000, state: "opened"}
             - assert: {from: 3, to: 4, total_deposit: 1000, balance: 1000, state: "opened"}
       - parallel:
-          name: "Enable MSs between node E and D (will be used later in the scenario)"
+          name: "Enable MSs between node 4 and 3 (will be used later in the scenario)"
           tasks:
             - store_channel_info: {from: 4, to: 3, key: "MS Test Channel"}
       - serial:
-          name: "Make 10 transfers from D to A"
+          name: "Make 10 transfers from 3 to 0"
           repeat: 10
           tasks:
             - transfer: {from: 3, to: 0, amount: 1}
-      - parallel:
-          name: "Assert after 10 payments from D to A"
+      - serial:
```

Comment:
when you have the parallel, then the `wait:` does not make sense. Without the wait, the assertion sometimes fails. Probably because of some latency of the balance update after the payment